documentation/src/crates/revm/journaled_state.md

```diff
@@ -136,6 +136,6 @@ This methodology allows Ethereum to maintain an internal record of accessed acco
 
 #### Rationale
 
-- Security: Previously, these opcodes were underpriced, making them susceptible to DoS attacks where an attacker would simply send transactions that access or call a large number of accounts. By increasing the gas costs, the EIP intends to mitigate these potential security risks.
+- Security: Previously, these opcodes were under-priced, making them susceptible to DoS attacks where an attacker would simply send transactions that access or call a large number of accounts. By increasing the gas costs, the EIP intends to mitigate these potential security risks.
```

Comment:
Isn't this one word?

Reply:
actually with hyphen it can be looked upon as  a compound adjective making it more readable